### PR TITLE
Removing old catch.hpp snapshot, Fixing DGtal on Apple M1

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: examples/doc-examples/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ set(DGTAL_DOC_EX_SRC
   labelledMapBestParameters
   )
 
+# # Make sure the compiler can find include files.
```
Comment (kerautret) [inline examples/doc-examples/CMakeLists.txt:8]: Cleaning ?
```suggestion
```

File: examples/doc-examples/CMakeLists.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ set(DGTAL_DOC_EX_SRC
...
+# include_directories (${PROJECT_SOURCE_DIR}/tests/)
```
Comment (kerautret) [inline examples/doc-examples/CMakeLists.txt:9]:
```suggestion
```

**Review by kerautret — COMMENTED** [review]
fine for mes thanks

**Review by kerautret — APPROVED** [review]

File: examples/doc-examples/CMakeLists.txt [hunk 3]
```diff
@@ -5,6 +5,7 @@ set(DGTAL_DOC_EX_SRC
   labelledMapBestParameters
   )
 
+# include_directories (${PROJECT_SOURCE_DIR}/tests/)
```
Comment (dcoeurjo) [inline examples/doc-examples/CMakeLists.txt:8]:
```suggestion
```

**Comment by dcoeurjo** [comment]
Thx @kerautret

**Pull request merged**